libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.stockPickerForm.valueChanges.subscribe(this.fetchQuote);
```

Comment:
As this.fetchQuote function is being called from subscribe function hence new execution context will be created for fetchQuote method and this keyword inside fetchQuote execution context won’t have access to the parent StockComponent’s functions as its scope is entirely different.